### ISSUE TITLE
Deprecated function: Creation of dynamic property PHPMailer::$is_html is deprecated

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/smtp#40]
I'm seeing the following:
```
Deprecated function: Creation of dynamic property PHPMailer::$is_html is deprecated in PHPMailer->IsHTML() (line 361 of /Users/jlampton/Sites/_backdrop/_contributions/smtp/includes/smtp.phpmailer.inc). 
```

Backdrop CMS:                          1.28.2
PHP version:                           8.2.24

**Comment by jenlampton** [comment]
I'm not seeing this anymore. Clsing for now.

**Comment by indigoxela** [comment]
> I'm not seeing this anymore. Clsing for now.

This has been fixed in #34 :wink: Updating to the latest version should be sufficient.
